docs/usage/faq.md

```diff
@@ -2,26 +2,44 @@
 
 ## How can I load external files in Pyodide?
 
-Two possible solutions are,
+In order to use external files in Pyodide, you should download and save them
+to the [Emscripten virtual file system](https://emscripten.org/docs/api_reference/Filesystem-API.html).
```

Comment:
I think this link is more likely to be confusing than helpful for most people.

Reply:
Yes, I agree.